_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:39
 msgid "You can test whether you are hearing your signal correctly by doing the following:"
-msgstr ""
+msgstr "So kannst du testen, ob du dein Signal richtig hörst:"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:41
 msgid "Run Jamulus and connect to a Server with a long ping time (greater than 200ms)"
-msgstr ""
+msgstr "Starte Jamulus und stelle eine Verbindung zu einem Server mit einer langen Ping-Zeit (mehr als 200 ms) her"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:42
 msgid "Clap your hands once (or play one short note on an instrument). You should hear the sound from the Jamulus Server come back, but significantly delayed."
-msgstr ""
+msgstr "Klatsche einmal in die Hände (oder spiele einen kurzen Ton auf einem Instrument). Du solltest hören, dass der Ton vom Jamulus Server zurückkommt, allerdings deutlich verzögert."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:43
 msgid "Click the \"Mute\" button under your Jamulus name/slider in the main Jamulus mixer window (**NOT** the \"Mute Myself\" button on the left)."
-msgstr ""
+msgstr "Klicke im Hauptfenster des Jamulus-Mixers auf die Schaltfläche \"MUTE\" unter deinem Jamulus-Namen/Slider (**NICHT** die Schaltfläche \"Stummschalten\" auf der linken Seite)."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:44
 msgid "Clap your hands again."
-msgstr ""
+msgstr "Klatsche noch einmal in die Hände."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:46
 msgid "If you hear two claps after step 2, or any claps after step 4, then you are **not** obeying Rule Number One - you have your local audio enabled and should disable it."
-msgstr ""
+msgstr "Wenn du nach Schritt 2 zwei klatschende Geräusche oder nach Schritt 4 ein Klatschen hörst, dann hälst du dich **nicht** an Regel Nummer eins - Du hast dein lokales Audio aktiviert und solltest es deaktivieren."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:48
 msgid "Exactly how you avoid listening to your direct signal will depend on your individual setup - your sound interface, mixing desk, headphone connection point, etc. For example, some audio interfaces have \"monitor\" buttons (turn these off), or similar options. **If you are still having problems**, try asking on the [forum](https://github.com/jamulussoftware/jamulus/discussions)."
-msgstr ""
+msgstr "Wie genau du vermeidest, dass du dein direktes Signal hörst, hängt von deiner individuellen Konfiguration ab - von deinem Audio-Interface, Mischpult, Kopfhöreranschluss usw. Einige Audioschnittstellen haben zum Beispiel \"Monitor\"-Tasten (schalte diese aus) oder ähnliche Optionen. **Wenn du immer noch Probleme hast, versuche im [Forum](https://github.com/jamulussoftware/jamulus/discussions) zu fragen."
```

Comment:
not "versuche im Forum zu fragen" but "frage im Forum"

Reply:
Corrected

Reply:
Corrected